docs/features/repo-pages.md

```diff
@@ -12,35 +12,47 @@ keywords:
 
 ## Accessing a Repository Page
 
-To view a repository page, follow these steps:
+There are a couple of ways to access a repository page:
 
-Open a web browser and navigate to the following URL structure:
+1. **From the Repository Insights page**: If you're already on the Repository Insights page, you can click on the repository name to view the repository page.
+2. **From the Search bar**: You can search for a repository by entering the repository name in the search bar. Once you find the repository you're looking for, click on the repository name to view the repository page.
+2. **Directly from the URL**: You can also access a repository page directly by entering the URL in your web browser: 
 
 `https://app.opensauced.pizza/s/{org}/{repo}`
 
 Replace `{org}` with the name of the organization that owns the repository you want to view. Replace `{repo}` with the name of the specific repository.
 
 For example, to view the `next.js` repository created by the `Vercel` organization, you would use the following URL:
 
-[https://app.opensauced.pizza/s/vercel/nextjs](https://app.opensauced.pizza/s/vercel/nextjs)
+[https://app.opensauced.pizza/s/vercel/nextjs](https://app.opensauced.pizza/s/vercel/next.js)
 
 Here's what that will look like:
 
-[![next.js repository page](../../static/img/repo-page.png)](https://app.opensauced.pizza/s/vercel/next.js) 
+[![next.js repository page](../../static/img/vercel-repo-page.png)](https://app.opensauced.pizza/s/vercel/next.js) 
 
 ## Understanding the Repository Page
 
-Here are some general things you might find on a repository page:
+### Overview Dashboard
+
+The charts on our repository pages offer a visual representation of several key metrics:
 
-- **Repository name and organization/user**: This is displayed prominently at the top of the page.
-- **Metrics**: This includes the number of stars and forks the repository has on GitHub.
+- **Issue Resolution Trends**: Our line graphs display the number of issues opened versus those closed within specified periods, offering insights into the effectiveness of problem resolution and how issue volume changes over time.
+- **Pull Request Dynamics**: Analyze pull request activity by the numbers of pull requests opened, closed, and merged. This provides an overview of collaboration efficiency and code review processes.
+- **Stars and Forks Over Time**: The number of stars and forks the project receives over time, illustrating the project's growth in popularity and its spread within the developer community.
+- **Lottery Factor**: This unique metric is visualized to highlight the impact of key contributors, showing how certain individuals' efforts disproportionately influence the project's progress. If 2 or fewer contributors account for 50% of the project's contributions, the lottery factor is high, indicating a potential risk if one of these contributors leaves the project.
```

Comment:
- **Question**: Because I don't quite understand how to interpret "disproportionately" here 😁. Can I interpret this as "significantly"? So, it's like:

   > This unique metric is visualized to highlight the impact of key contributors, demonstrating how the efforts of certain individuals significantly influence the project's progress.

- Clarity and link to glossary suggestion:

```suggestion
- **[Lottery Factor](../welcome/glossary.md#lottery-factor)**: This unique metric is visualized to highlight the impact of key contributors, showing how certain individuals' efforts disproportionately influence the project's progress. The lottery factor is high if 2 or fewer contributors account for 50% of the project's contributions. It indicates a potential risk to the project's sustainability if one of these contributors leaves the project.
```